src/test/resources/rca/rca.conf

```diff
@@ -41,5 +41,7 @@
     // How often the sqlite file be repeated in seconds. This file contains RCAs and therefore rotating it too frequently
     // might not be as fruitful as there might not be any data.
     "rotation-period-seconds": 21600
-  }
+  },
+
+  "muted-rcas": "rca1, rca2"
```

Comment:
Can we also have a couple of negative test cases and one where the comma separated rcas don't have spaces between them ?

Reply:
We should add a test at the graph creation level where no Graph node matches the names of the Rca being asked to be muted and And one where a graph node is successfully muted

Reply:
Agreed, the test I am writing currently and will cover the happy cases and otherwise.